scikits/odes/sundials/cvode.pyx

```diff
@@ -864,6 +887,8 @@ cdef class CVODE:
 
             self.options['interruptfn'] = fn
 
+            self._old_api = False
+
```

Comment:
Yeah, I haven't added documentation about the new api, but I would add a note that using things that require the new api (interruptfn or its successor) will return via the new api. I should probably add an option to force the old api.